src/Tizen.Content.Download/Tizen.Content.Download/Request.cs

```diff
@@ -843,6 +844,37 @@ private void UnregisterProgressChangedEvent()
                 DownloadErrorFactory.ThrowException(ret, "Unsetting ProgressChanged callback failed");
             }
         }
+
+        /// <summary>
+        /// Enabled state of the cache feature.
+        /// </summary>
+        /// <since_tizen> 7.5 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/download</privilege>
+        /// <exception cref="ArgumentException">Thrown when it is failed due to an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when it is failed due to an invalid operation.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when a permission is denied.</exception>
+        /// <exception cref="NotSupportedException">Thrown when not supported.</exception>
+        public bool CacheEnable
```

Comment:
Please consider changing the name to "UsesCache", "IsCacheEnabled", or similar that is better for boolean property. EnableCache() would be good name for method, UsesCache is better for property, so the the code be like `request.UsesCache = true`. See note about booleans at  [Member naming convention](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-properties)

Reply:
Changed to UsesCache.